### DISCUSSION TITLE
How to solve this strange BTE problem

### DISCUSSION THREAD

**Discussion by oscillatortyy** [lululxvi/deepxde#1021] [Q&A]
Hello ,community！
Dear Lulu, thank you so much for your contribution to this library @lululxvi 
I recently learned to use this library, and now I have a rather strange problem to solve 
I'm going to solve a BTE.

![LKR EAARC_7YO`JY9FA%@Q](https://user-images.githubusercontent.com/117383363/200809807-70175243-2204-4109-84a5-edf6e50655dd.png)

**We don't have a reference solution, but what we have is the values of the grid points z, kz and the corresponding velocity   vn, force  Fn, term of scattering Q, distribution function  fn and  fneq is used for Dirichlet boundary conditions**

![8A6}{17A7ZUIH `F{@%3XU](https://user-images.githubusercontent.com/117383363/200810077-d988b95f-c061-4e3b-ab9a-de5215448171.png)

![4P{G JF`SW$M`C@5PX7IFYH](https://user-images.githubusercontent.com/117383363/200810115-b2fe162c-eea6-4908-a8bd-3d59e442dd94.png)

So I need to use these grid points to predict the distribution function fn through a PINN，
Now I have a few questions：
1.What should I do with vn,Fn,Q,  can it be used as output？
2.**What to do with these grid points as training data**？Do I need to use anchors？Does PointSetBC have anything to do with this?
3.How to define boundary conditions？
Thank you very much and look forward to receiving your early reply!

**Comment by lululxvi** [comment]
1. Yes, seems a good way.
2. Yes, you can use anchors for PDE.
3. Use PointSetBC for BC.
